src/shared_modules/dbsync/include/dbsync.h

```diff
@@ -161,7 +161,7 @@ EXPORTED int dbsync_sync_row(const DBSYNC_HANDLE handle,
  * @brief Select data, based in \p json_data_input data, from the database table.
  *
  * @param handle          Handle assigned as part of the \ref dbsync_create method().
- * @param js_data_input   JSON with table name, fields and filters to apply in the query.
+ * @param js_data_input   JSON with table name, fields, filters and options to apply in the query.
```

Comment:
Where are these filters documented? I should modify whatever documentation we have to reflect these changes.